spec/models/user_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user_1) { FactoryGirl.create(:user) }
+
+  it 'is valid with a name, email, and password' do
+    expect(user_1).to be_valid
+  end
+
+  it 'is invalid without a name' do
+    user_1.name = nil
+    user_1.valid?
+    expect(user_1.valid?).to be(false)
+    expect(user_1.errors[:name]).to include("can't be blank")
+  end
+
+  it 'is invalid without an email address' do
+    user_1.email = nil
+    user_1.valid?
```

Comment:
now you can remove `user_1.valid?`, `expect(user_1.valid?).to be(false)` should be more than fine. So your test should look like 
`user_1.email = nil 

 expect(user_1.valid?).to be(false)

 expect(user_1.errors[:email]).to include("can't be blank")`